### ISSUE TITLE
AJAX Background Jobs Fail After a Period of Inactivity

### ISSUE THREAD

**Issue by Torqu3Wr3nch** [nextcloud/docker#1442]
In short, I'm seeing an issue where AJAX-triggered background jobs are working perfectly and then, after a period of inactivity overnight, the following morning I'll find that Nextcloud background jobs are no longer running.

I'm able to get AJAX to work again but it requires running the cron job manually with ```php /var/www/html/cron.php``` (and then switching back to AJAX) before AJAX-triggered background jobs will start working again.

Additional details, including a screenshot can be found here: https://forums.engineerworkshop.com/t/onlyoffice-documents-not-saved-in-nextcloud-how-to-recover/308/3?u=torquewrench

I'm opening this up under Nextcloud/docker because I'm running the official Nextcloud Docker container, but it may very well be a core Nextcloud issue. Let me know if I should open up the issue there instead.

**Comment by J0WI** [comment]
AJAX jobs are not as powerful as commands on the console. You can try to tweak `max_execution_time` or `memory_limit` but the recommended method is to use cron: https://docs.nextcloud.com/server/stable/admin_manual/configuration_server/background_jobs_configuration.html

**Comment by keponk** [comment]
@J0WI shouldn't the recommended way be already part of the docker image?

**Comment by J0WI** [comment]
It is included, see https://github.com/nextcloud/docker/issues/627 for examples.

**Comment by keponk** [comment]
you're correct, I meant to say that the recommended way be the default. Since the image is maintained by NC, everyone would expect all the defaults are set to the recommended settings.

**Comment by J0WI** [comment]
We inherit the defaults from NC
